po/es.po

```diff
@@ -152,27 +152,27 @@ msgstr ""
 #, c-format
 msgid "git apply: bad git-diff - expected /dev/null, got %s on line %d"
 msgstr ""
-"git apply: mal git-diff - se esperaba /dev/null, se encontró %s en la línea "
-"%d"
+"git apply: git-diff erróneo - se esperaba /dev/null, se encontró %s en "
+"lalínea %d"
```

Comment:
Hi @ChrisADR 
The replacement of "la línea" by "lalínea", without spaces, it's that correct?

Reply:
Hi @CGarces, nope it's not, I'll fix it on my repo and will be merged in the next release, thanks for pointing this!

Reply:
I just doube checked, said diff is not present in the most recent version :/ I'll still consider changing the "mal" with the "erróneo", sounds more 'professional' :) thanks!